clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -745,7 +745,17 @@ LValue CIRGenFunction::buildUnaryOpLValue(const UnaryOperator *E) {
   }
   case UO_PreInc:
   case UO_PreDec: {
```

Comment:
Looks like you got a bit confused while using git rebase, `UO_PreDec` handling and `unary.cpp` are part of another PR that already landed, this needs to be fixed as well.

Reply:
I think this is because I rebased against a commit that got erased/force-pushed. I will rebase again with the local patch I have.